src/accelerate/tracking.py

```diff
@@ -500,7 +500,7 @@ def store_init_configuration(self, values: dict):
                     f'Trainer is attempting to log a value of "{value}" for key "{name}" as a parameter. MLflow\'s'
                     f" log_param() only accepts values no longer than {mlflow.utils.validation.MAX_PARAM_VAL_LENGTH} characters so we dropped this attribute."
                 )
-                del values[name]
+                values.pop(name)
```

Comment:
While this is equivalent, `del values[names]` is explicit about what it's doing. `values.pop(name)` is not, so this should really be reverted.